### ISSUE TITLE
Crews & Units - Add ability to add multiple products to a single bulker

### ISSUE THREAD

**Issue by JasonSchneider19** [Sanjel-Energy-Services/eService_WorkItems#238]
@awangatsanjel
Feedback:
In eservice, under the page Units and personnel, in the storage section in the service ticket. It would be nice to be able to input the same storage unit number more than once. I understand the issue if the same P-Tank was able to be put in but if there was a way to limit it to bulk trailers that would awesome. For example; today I had a bulk trailer that had one pod with Visweep DM and another pod that had Productionmix LW. I was unable to differentiate between the two sources on the storage part. This section directly computes to the post job. I had to "make up" a storage tank for the product to show in my post job so that it read correctly.

Update eService with product haul data:
![image](https://github.com/user-attachments/assets/f2829fe4-4868-4605-9a1f-5a7511b91015)
When product haul has multiple product hauls, have it shown in eService so that the Post Job is updated properly.

**Comment by awangatsanjel** [comment]
@JasonSchneider19 

I am seeing the original idea was for the "Sanjel Storage Equipment" section. You may mis-read it. I think it talks about using a bulk trailer as storage on site, the pod content information should be visible.

We will need to make the change in Post Job Report in eService Express to align with the product haul changes. The current eService UI was legacy, it cared more about the equipment. Now we can care further about the content. The pod concept has been implemented.

Please correct if my understanding is wrong.

**Comment by JasonSchneider19** [comment]
@awangatsanjel 
You are correct, and yes I believe that we have all the information in the way that we are moving the product. Just really making it visible - in eService and post job.

**Comment by awangatsanjel** [comment]
@lukeyang1972 @belazh @censinli 请进行一下预估

**Comment by awangatsanjel** [comment]
Architect Notes:

We need to make following changes to satisfy this requirement for future.

1.   In eService, make following changes to "Sanjel Storage Equipment"
- Add bulker unit types in Type list
- Add Pod Index, Blend fields in the record structure.
- This section data is populated by eServiceOnline operations
- This section data will be carried over to Service Report when job is created from call sheet as it is.

2.  In eServiceOnline, update the product haul and bin inforamtion changes to call sheet "Sanjel Storage Equipment".
- If a product haul is scheduled to "Go with crew", the bulker cotent information needs to be populated by per pod.
- The bins assigned to the rig was populated in this section before, but only bin type and number. We will need to populate the pod index,  blend and tonnage when the blend in bin is updated (product haul offload, empty bin by user or update from other integration. 

3. In eServiceExpress, Storage Volume section in Post Job needs following change.
- Add PodIndex
- Populate all data from "Sanjel Storage Equipment" in Service Report when the post job is created from the job package.
- Any change to these data will be sent to server for the post job report

@JasonSchneider19 Please verify above descriptions.

@censinli This task can be moved ahead once Jason confirmed

cc @belazh @zhangyuan5

**Comment by belazh** [comment]
- [ ] Add Pod Index, Blend Info in BinSection in Sanjel Storage Equipment
- [ ] Update the product haul and bin inforamtion changes to CallSheet "Sanjel Storage Equipment"
- [ ] Populate all data from "Sanjel Storage Equipment" in Service Report when the Post Job is created from the Job Package
